src/uv-common.c

```diff
@@ -155,6 +155,82 @@ static const char* uv__unknown_err_code(int err) {
   return copy != NULL ? copy : "Unknown system error";
 }
 
+int uv_translate_posix_error(int sys_errno) {
+  switch (sys_errno) {
```

Comment:
I did it this way to that if the function is passed something that doesn't make sense in libuv (e.g. ECHILD), then it will return UV_UNKNOWN instead of some garbage value. Is that a non-issue?

The other reason was that the function written this way would also work on Windows. My understanding is that it needed to work on both Unix and Windows, and on Windows it would need to be written like this anyways (unless I'm missing something?), so might as well just write it once for both.